src/themis/secure_comparator.c

```diff
@@ -38,7 +38,7 @@ static themis_status_t ed_sign(uint8_t pos, const uint8_t *scalar, uint8_t *sign
 {
 	uint8_t r[ED25519_GE_LENGTH];
 	ge_p3 R;
-	uint8_t k[64];
+	uint8_t k[64]; //temp buffer for group elements and hash
```

Comment:
Although we all praise Bernstein for his cryptographical efforts, we might add better style and readability ourselves, don't we?